SwSpotify/spotify.py

```diff
@@ -112,13 +114,31 @@ def get_info_mac():
     return a[3], a[1]
 
 
+def get_info_web():
+    """
+    Uses chrome extension to get the data
+    """
+    sys.stderr = open(os.devnull, "w")
+    result = spotify_web.run()
+    sys.stderr = sys.__stderr__
+
+    if not result:
+        raise SpotifyClosed
+
+    track, artist = result
+    return track, artist
+
+
 def current():
-    if sys.platform.startswith("win"):
-        return get_info_windows()
-    elif sys.platform.startswith("darwin"):
-        return get_info_mac()
-    else:
-        return get_info_linux()
+    try:
+        if sys.platform.startswith("win"):
+            return get_info_windows()
+        elif sys.platform.startswith("darwin"):
+            return get_info_mac()
+        else:
+            return get_info_linux()
+    except SpotifyClosed:
```

Comment:
You probably want to catch SpotifyNotRunning since if the desktop app is open but the music is playing on the Web Player then SpotifyPaused would be raised, not Closed